projects/storefrontlib/shared/components/card/card.component.ts

```diff
@@ -7,14 +7,17 @@
 import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { ICON_TYPE } from '../../../cms-components/misc/icon/icon.model';
 
-export interface CardAction {
```

Comment:
Changing or removing interfaces is considered a [breaking change](https://wiki.one.int.sap/wiki/display/spar/Spartacus+Onboarding#:~:text=Changing%20or%20removing%20interface%20properties.).